stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/QuotedMessage.kt

```diff
@@ -156,11 +158,13 @@ internal fun DefaultQuotedMessageTrailingContent(
 @Composable
 public fun RowScope.DefaultQuotedMessageCenterContent(
```

Comment:
Not  super important since it's just default content, but we could add a param doc here for consistency.

Reply:
didn't see this one, thanks